docs/reference/setup/logging-config.asciidoc

```diff
@@ -139,67 +139,51 @@ documentation].
 [[configuring-logging-levels]]
 === Configuring logging levels
 
-There are four ways to configuring logging levels, each having situations in which they are appropriate to use.
-
-1. Via the command-line: `-E <name of logging hierarchy>=<level>` (e.g.,
-   `-E logger.org.elasticsearch.transport=trace`). This is most appropriate when
-   you are temporarily debugging a problem on a single node (for example, a
-   problem with startup, or during development).
```

Comment:
I intentionally dropped the command line example here. This seems to be primarily for development and wouldn't be used much in production. Let me know if that's wrong or if we need to include it for another reason.